test/test_tensordict.py

```diff
@@ -1757,6 +1757,34 @@ def test_split(self, td_name, device, performer):
                                 expected_inner_tensor_size
                             )
 
+    def test_pop(self, td_name, device):
+        td = getattr(self, td_name)(device)
+        assert "a" in td.keys()
+        a = td["a"].clone()
+        out = td.pop("a")
+        assert (out == a).all()
+        assert "a" not in td.keys()
+
+        default = torch.Tensor([0])
```

Comment:
having a string was cool actually! We can keep that and `assert out != default` as it was before

Reply:
yes, it was.. but if I use string in get function I get this error:
ValueError: default should be None or a Tensor instance, got some value

Reply:
Ah right, well we could just do 
`default = torch.zeros_like(a)` then